duckduckgo/macos-browser
# Update find in page implementation for input/textarea support

Task/Issue URL: https://app.asana.com/0/72649045549333/1200260792509443/f
Tech Design URL: https://app.asana.com/0/72649045549333/1200308960321051/f
CC:

**Description**:
Update to the latest version of find in page with support for textarea and input matching -- https://github.com/more-duckduckgo-org/duckduckgo-find-in-page

**Steps to test this PR**:
1. See ongoing product review for browser build -- https://app.asana.com/0/72649045549333/1200422750443843/f


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
